src/azure-cli/azure/cli/command_modules/cdn/custom.py

```diff
@@ -71,6 +91,315 @@ def update_endpoint(instance,
     return params
 
 
+# pylint: disable=too-many-return-statements
+def create_condition(match_variable=None, operator=None, match_values=None,
```

Comment:
Any chance to remove the `# pylint: disable=too-many-return-statements` or reduce the numbers of lines?

Reply:
I can fix the return statement but I don't think I have a way to reduce the number of lines